.github/workflows/wrapper.yml

```diff
@@ -0,0 +1,23 @@
+name: Validate Gradle Wrapper
+on: [pull_request]
+defaults:
+  run:
+    shell: bash
+    working-directory: gradle/wrapper
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check Gradle Wrapper
```

Comment:
Gradle publishes an action for verifying jars via https://github.com/gradle/wrapper-validation-action, may be we should give it a shot.

Reply:
```
name: Validate Gradle Wrapper
on:
  pull_request:
    branches:
      - main

jobs:
  validation:
    name: Validation
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: gradle/wrapper-validation-action@v1
```

Reply:
Thanks for finding this, updated!

Reply:
```
2s
Run gradle/wrapper-validation-action@v1
✓ Found known Gradle Wrapper JAR files:
  33ad4583fd7ee156f533778736fa1b4940bd83b433934d1cc4e9f608e99a6a89 gradle/wrapper/gradle-wrapper.jar
```

Reply:
Nice!